unit-tests/dds/test-librs-formats-converison.py

```diff
@@ -0,0 +1,178 @@
+# License: Apache 2.0. See LICENSE file in root directory.
+# Copyright(c) 2023 Intel Corporation. All Rights Reserved.
+
+#test:donotrun:!dds
+
+from rspy import log, test
+import pyrealsense2 as rs
+import dds
+
+rs.log_to_console( rs.log_severity.debug )
+log.nested = 'C  '
+
+context = rs.context( '{"dds-domain":123,"dds-participant-name":"test-formats-conversion"}' )
+only_sw_devices = int(rs.product_line.sw_only) | int(rs.product_line.any_intel)
+
+import os.path
+cwd = os.path.dirname(os.path.realpath(__file__))
+remote_script = os.path.join( cwd, 'formats-conversion-server.py' )
+with test.remote( remote_script, nested_indent="  S" ) as remote:
+    remote.wait_until_ready()
+
+    # Tests setup
+    remote.run( 'create_server()' )
+    n_devs = 0
+    for dev in dds.wait_for_devices( context, only_sw_devices ):
+        n_devs += 1
+    test.check_equal( n_devs, 1 )
```

Comment:
I think you may want to put this setup under a `test.closure` with `on_fail=test.ABORT`?

Reply:
Added